### ISSUE TITLE
[TESTING]: STEM Standalone - All accordions SHOULD be opened and tested with axe in E2E suite

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#12344]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The accordions on the review page should be evaluated for automated accessibility, both opened and closed. This will give us better test coverage for accessibility.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Test(s) written that run `axeCheck()` when accordions are closed
- [ ] Test(s) written that run `axeCheck()` when accordions are opened
- [ ] 0 axe violations are thrown when tests are run

## Environment

* https://staging.va.gov/education/apply-for-education-benefits/application/10203/review-and-submit

**Comment by desireeturner** [comment]
We are planning to work this issue after we've completed our initial launch.

**Comment by 1Copenut** [comment]
> We are planning to work this issue after we've completed our initial launch.

Not a problem. @U-DON did a lot of work with the Cypress form tester, so it's possible this might already be happening. If not, getting this hidden content under test is a good post-launch item.

**Comment by hughes-dustin** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/14232

branch: bah-12344

**Comment by sonntag-adam** [comment]
Tests pass.

**Comment by sonntag-adam** [comment]
Merged, closing story.

**Comment by sonntag-adam** [comment]
@1Copenut tests are merged in

**Comment by 1Copenut** [comment]
@sonntag-adam Excellent, thank you. I'll close this issue.
